app/components/provider_interface/application_choice_header_component.rb

```diff
@@ -36,7 +36,7 @@ def sub_navigation_items
       sub_navigation_items = [application_navigation_item]
 
       sub_navigation_items.push(offer_navigation_item) if offer_present?
-      sub_navigation_items.push(references_navigation_item) if FeatureFlag.active?(:new_references_flow_providers)
+      sub_navigation_items.push(references_navigation_item) if FeatureFlag.active?(:new_references_flow_providers) && !application_choice.status.to_sym.in?(ApplicationStateChange::UNSUCCESSFUL_END_STATES)
```

Comment:
would be nice to extract `!application_choice.status.to_sym.in?(ApplicationStateChange::UNSUCCESSFUL_END_STATES)` to some appropriately named method, eg `application_unsuccessful?`

Reply:
Done! ♻️